src/styles/block/is-active.scss

```diff
@@ -0,0 +1,14 @@
+.is-active {
+  color: $header-color;
+
+  &::after {
+    content: '';
+    display: block;
+    position: relative;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), 'position' property should come before 'display'. Please reorder the properties accordingly.